pkg/helm/helm.go

```diff
@@ -50,19 +50,14 @@ type TemplateRenderer interface {
 // NewHelmRenderer creates a new helm renderer with the given parameters and returns an interface to it.
 // The format of helmBaseDir and profile strings determines the type of helm renderer returned (compiled-in, file,
 // HTTP etc.)
-func NewHelmRenderer(chartsRootDir, helmBaseDir, profile, componentName, namespace string) (TemplateRenderer, error) {
-	globalValues, err := ReadValuesYAML(profile)
```

Comment:
I think removing the `profile` here will result in plenty of empty fields, the chart renderer will fail unless we manually override all these fields, please check the prow test job: https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_operator/161/operator-test/88